WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -102,4 +102,14 @@ def competitions_json_for_markers(competitions)
       }
     end.to_json.html_safe
   end
+
+  def championship_select_fields
+    [
+      ["World", "world"],
+      ["", ""],
```

Comment:
I'm not in love with how this looks:

![image](https://user-images.githubusercontent.com/277474/28848398-a85f4f54-76c7-11e7-9764-f1ef8ab17e66.png)


What about using `optgroups` here instead to achieve a look like this:

![image](https://user-images.githubusercontent.com/277474/28848271-2b74a1ec-76c7-11e7-9a91-675053578680.png)

Unfortunately, it's not immediately obvious to me how to achieve this with simpleform. Here are the [relevant docs](https://github.com/plataformatec/simple_form#collections), but that looks like it might get weird, as these aren't active record types.  [This SO answer](https://stackoverflow.com/a/28393641) might be more useful.